versions/3.0.4.md

```diff
@@ -1765,9 +1746,11 @@ Note also that `Content-Transfer-Encoding` is deprecated for `multipart/form-dat
 Using `format: byte` for a multipart field is equivalent to setting `Content-Transfer-Encoding: base64`.
```

Comment:
I'm confused by this. If `Content-Transfer-Encoding` is deprecated and a multipart field with `format: byte` is equivalent to setting `Content-Transfer-Encoding: base64`, doesn't this imply that a multipart field with `format: byte` is essentially deprecated?

I guess what I'm missing here is why `Content-Transfer-Encoding` was deprecated. It seems like this effectively disallows any encoding of individual fields in a multipart body, which seems undesirable. I'm sure this is just confusion on my part but would appreciate if someone could clear this up for me.

Reply:
I agree that we are obligated to follow the RFCs, which is exactly why I think we need to clearly state that `format: byte` is deprecated for a property in a multipart/form-data body. 

@darrelmiller Would appreciate your thoughts on this.

Reply:
Deprecated in OAS is not the same as deprecated in RFC7578, and the text I wrote _already_ makes it clear that the _usage_ is deprecated by RFC7578 in this context.